FaunaDB.Client.Test/TestCase.cs

```diff
@@ -50,6 +53,15 @@ private async Task SetUpAsync()
             faunaSecret = env("FAUNA_ROOT_KEY", "secret");
             faunaEndpoint = port != "443" ? $"{scheme}://{domain}:{port}/" : $"{scheme}://{domain}/";
             rootClient = new FaunaClient(secret: faunaSecret, endpoint: faunaEndpoint);
+            rootClientWithCustomHeaders = new FaunaClient(
+                secret: faunaSecret,
+                endpoint: faunaEndpoint,
+                customHeaders: new Dictionary<string, string>
+                {
+                    { "test-header-1", "test-value-1" },
+                    { "test-header-2", "test-value-2" },
+                }
+            );
 
```

Comment:
what do you think on moving the client creation only to the test itself? 
I don't think is necessary to have them as  members on the base class if we are only going to use once, to test the headers

Reply:
do we need both `rootClientWithCustomHeaders` and `clientWithCustomHeaders` ? I didn't find any use for `rootClientWithCustomHeaders`

Reply:
makes sense, moved it to the test